### ISSUE TITLE
Compilation warning

### ISSUE THREAD

**Issue by yanivshaked** [SachinGanesh/screenshot#47]
Note the following compilation warning; delay is not nullable so the `??` operation is redundant.
```
/C:/src/flutter/.pub-cache/hosted/pub.dartlang.org/screenshot-1.1.0/lib/screenshot.dart:45:31: Warning: Operand of null-aware operation '??' has type 'Duration' which excludes null.
 - 'Duration' is from 'dart:core'.
    return new Future.delayed(delay ?? Duration(milliseconds: 20), () async {
```

**Comment by armandojimenez** [comment]
Same issue

**Comment by yanivshaked** [comment]
The accepted PR #48 fixes this issue, @SachinGanesh can you please release a new version?

**Comment by SachinGanesh** [comment]
Fixed in version 1.2.1
